### PR TITLE
Improvements for #2

### PR DESCRIPTION
**Pull request by luoe**

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -3,7 +3,8 @@
 #General Informations
 $CONFIG_CONTACT = 'fsi@fsi.uni-tuebingen.de';
 $CONFIG_TERM = 'WS 20/21'; # example: WS 19/20 or SS 20
-$fp = "../../eei-registration/"; #File Prefix
+// see https://www.php.net/manual/en/language.constants.predefined.php
+$fp = "__DIR__/../eei-registration/"; #File Prefix
```
Comment (primeos) [inline config.php:7]: I think this needs to be:
```suggestion
$fp = __DIR__ . "/../eei-registration/"; #File Prefix
```

(my example from the last review was bad)

File: registration.php [hunk 2]
```diff
@@ -20,7 +22,7 @@ function registerForOfflineEvent($event){
 
         // already registered
         if(strpos(file_get_contents($file), $mail) !== false || strpos(file_get_contents($file), $phone) !== false){
-            echo "<div class='block info'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
+            echo "<div class='block warning'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
```
Comment (primeos) [inline registration.php:25]: Does this work? There is indeed already a definition for `error`:
```
css/style.css:.error {
css/style.css-    color: red;
css/style.css-    border: 1px solid red;
css/style.css-    border-radius: 2px;
css/style.css-    margin: 5px 0;
css/style.css-    padding: 2px;
css/style.css-}
```

But I didn't find a class definition for `warning`.

File: registration.php [hunk 3]
```diff
@@ -71,7 +73,7 @@ function registerForOnlineEvent($event){
 
         // already registered
         if(strpos(file_get_contents($file), $mail) !== false){
-            echo "<div class='block info'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
+            echo "<div class='block warning'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
```
Comment (primeos) [inline registration.php:76]: See comment above.

**Review by primeos — CHANGES_REQUESTED** [review]

**Comment by luoe** [comment]
Ok, thanks, fixed it :)

File: registration.php [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-    require_once('__DIR__/config.php');
+    require_once(__DIR__ . '/config.php');
```
Comment (primeos) [inline registration.php:3]: Oh, I missed that change. I don't think there's any point in using `__DIR__` here (for includes in general) but of course there's also nothing wrong with it (technically).

**Review by primeos — APPROVED** [review]
LGTM now, thanks :)

**Pull request merged**